docs/REST API/rest-api.md

```diff
@@ -9483,10 +9542,12 @@ Update a package to install on macOS, Windows, or Linux (Ubuntu) hosts.
 | self_service | boolean | form | Whether this is optional self-service software that can be installed by the end user. |
 | labels_include_any        | array     | form | Target hosts that have any label in the array. Only one of either `labels_include_any` or `labels_exclude_any` can be specified. |
 | labels_exclude_any | array | form | Target hosts that don't have any label in the array. |
+| automatic_install | boolean | form | Create a policy that triggers a software install only on hosts missing the software. |
+| automatic_update | boolean | form | Create a policy that triggers a software install on all hosts running an older version. |
 
 Only one of `labels_include_any` or `labels_exclude_any` can be specified. If neither are specified, all hosts are targeted.
 
-> Changes to the installer package will reset installation counts. Changes to any field other than `self_service` will cancel pending installs for the old package.
+Changes to `software` (the package) will reset `installed`, `pending`, and `failed` counts. Changes to any field other than `self_service` will cancel pending installs and uninstalls.
```

Comment:
```suggestion
Changes to any field other than `self_service` will cancel pending installs and uninstalls. Changes to `software` (the package) will reset `installed` and `failed` counts.
```

Suggesting because we're cancelling pending installs, not just resetting the count.